### ISSUE TITLE
The loss of training verification set on 3080ti graphics card is always 0(To be more precise, is empty.)

### ISSUE THREAD

**Issue by StudyingLover** [ultralytics/yolov5#12291]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Training

### Bug

Hello,I am a loyal user of yolov5. Recently I was doing an experiment using yolov5 for target detection. I use a 3080ti cloud server and a 4060 personal computer for training. In addition, I made modifications to the model. The modified files are as follows.I named it small.yaml.
```
# parameters
nc: 1  # number of classes
depth_multiple: 0.33  # model depth multiple
width_multiple: 0.50  # layer channel multiple
 
# anchors
anchors:
  - [5,6, 8,14, 15,11]  #4
  - [10,13, 16,30, 33,23]  # P3/8
  - [30,61, 62,45, 59,119]  # P4/16
  - [116,90, 156,198, 373,326]  # P5/32
  
backbone:
  # [from, number, module, args]
  [[-1, 1, Focus, [64, 3]],  # 0-P1/2
   [-1, 1, Conv, [128, 3, 2]],  # 1-P2/4
   [-1, 3, C3, [128]],
   [-1, 1, Conv, [256, 3, 2]],  # 3-P3/8
   [-1, 9, C3, [256]],
   [-1, 1, Conv, [512, 3, 2]],  # 5-P4/16
   [-1, 9, C3, [512]],
   [-1, 1, Conv, [1024, 3, 2]],  # 7-P5/32
   [-1, 1, SPP, [1024, [5, 9, 13]]],
   [-1, 3, C3, [1024, False]],  # 9
  ]

# YOLOv5 head
head:
  [[-1, 1, Conv, [512, 1, 1]],#20*20
   [-1, 1, nn.Upsample, [None, 2, 'nearest']],
   [[-1, 6], 1, Concat, [1]],  # cat backbone P4
   [-1, 3, C3, [512, False]],  # 13

   [-1, 1, Conv, [512, 1, 1]], #40*40 14
   [-1, 1, nn.Upsample, [None, 2, 'nearest']],
   [[-1, 4], 1, Concat, [1]],  # cat backbone P3   80*80
   [-1, 3, C3, [512, False]],  # 17 (P3/8-small)  80*80

   [-1, 1, Conv, [256, 1, 1]],
   [-1, 1, nn.Upsample, [None, 2, 'nearest']],
   [[-1, 2], 1, Concat, [1]],  # cat backbone P3
   [-1, 3, C3, [256, False]],  # 17 (P3/8-small)21

   [-1, 1, Conv, [256, 3, 2]],
   [[-1, 18], 1, Concat, [1]],  # cat head P4
   [-1, 3, C3, [256, False]],  # 20 (P4/16-medium)24

   [-1, 1, Conv, [256, 3, 2]],  #22   80*80
   [[-1, 14], 1, Concat, [1]], #23 80*80
   [-1, 3, C3, [512, False]], #24 80*80

   [-1, 1, Conv, [512, 3, 2]],
   [[-1, 10], 1, Concat, [1]],  # cat head P5
   [-1, 3, C3, [1024, False]],  # 23 (P5/32-large)

   [[21, 24, 27,30], 1, Detect, [nc, anchors]],  # Detect(P3, P4, P5)
  ]
```
I also created a data.yaml and placed the information of my dataset
```
# Train/val/test sets as 1) dir: path/to/imgs, 2) file: path/to/imgs.txt, or 3) list: [path/to/imgs1, path/to/imgs2, ..]
path: /root/data/data  # dataset root dir
train: train  # train images (relative to 'path') 128 images
val: val  # val images (relative to 'path') 128 images
test: val # test images (optional)

# Classes (80 COCO classes)
names:
  0: drone
```
I ran the following two commands on the server to train the model
```
python train.py --data data.yaml --epochs 10 --batch-size -1 --cfg small.yaml
```
and
```
python train.py --data data.yaml --epochs 10 --batch-size -1
```
I got result.png as follows
![results](https://github.com/ultralytics/yolov5/assets/68844494/d416c5ef-4aac-4594-a417-49c62caa3604)

![results](https://github.com/ultralytics/yolov5/assets/68844494/466eddb8-1ebe-4885-ab7f-d564e3475d87)

Maybe you think this is because the training times are too few. I will provide you with the training results on 4090 below with samll.yaml.
![results](https://github.com/ultralytics/yolov5/assets/68844494/d8baed62-637d-42c2-b06d-722321556a6f)

In order to avoid this, I used 3080 to train the same number of times, with and without small.yaml
![results](https://github.com/ultralytics/yolov5/assets/68844494/566daed7-1a5a-4bf5-b422-771f04630ec6)
![results](https://github.com/ultralytics/yolov5/assets/68844494/410e3ef7-29a6-4e35-9941-7a7010873c3e)

### Environment

4090
Python Version: 3.11.3
Torch Version: 2.0.1+cu117
System Version: Linux
YOLOV5: lealatest

3080ti
Python Version: 3.8.10
Torch Version: 1.9.0+cu111
System Version: Linux
YOLOV5:  commit hash is `50ff6eee31c72fe88bdd35fc7299b201cce0e9a6`

3080
Python Version: 3.8.10
Torch Version: 1.9.0+cu111
System Version: Linux
YOLOV5: lealatest

### Minimal Reproducible Example

I made modifications to the model. The modified files are as follows.I named it small.yaml.
```
# parameters
nc: 1  # number of classes
depth_multiple: 0.33  # model depth multiple
width_multiple: 0.50  # layer channel multiple
 
# anchors
anchors:
  - [5,6, 8,14, 15,11]  #4
  - [10,13, 16,30, 33,23]  # P3/8
  - [30,61, 62,45, 59,119]  # P4/16
  - [116,90, 156,198, 373,326]  # P5/32
  
backbone:
  # [from, number, module, args]
  [[-1, 1, Focus, [64, 3]],  # 0-P1/2
   [-1, 1, Conv, [128, 3, 2]],  # 1-P2/4
   [-1, 3, C3, [128]],
   [-1, 1, Conv, [256, 3, 2]],  # 3-P3/8
   [-1, 9, C3, [256]],
   [-1, 1, Conv, [512, 3, 2]],  # 5-P4/16
   [-1, 9, C3, [512]],
   [-1, 1, Conv, [1024, 3, 2]],  # 7-P5/32
   [-1, 1, SPP, [1024, [5, 9, 13]]],
   [-1, 3, C3, [1024, False]],  # 9
  ]

# YOLOv5 head
head:
  [[-1, 1, Conv, [512, 1, 1]],#20*20
   [-1, 1, nn.Upsample, [None, 2, 'nearest']],
   [[-1, 6], 1, Concat, [1]],  # cat backbone P4
   [-1, 3, C3, [512, False]],  # 13

   [-1, 1, Conv, [512, 1, 1]], #40*40 14
   [-1, 1, nn.Upsample, [None, 2, 'nearest']],
   [[-1, 4], 1, Concat, [1]],  # cat backbone P3   80*80
   [-1, 3, C3, [512, False]],  # 17 (P3/8-small)  80*80

   [-1, 1, Conv, [256, 1, 1]],
   [-1, 1, nn.Upsample, [None, 2, 'nearest']],
   [[-1, 2], 1, Concat, [1]],  # cat backbone P3
   [-1, 3, C3, [256, False]],  # 17 (P3/8-small)21

   [-1, 1, Conv, [256, 3, 2]],
   [[-1, 18], 1, Concat, [1]],  # cat head P4
   [-1, 3, C3, [256, False]],  # 20 (P4/16-medium)24

   [-1, 1, Conv, [256, 3, 2]],  #22   80*80
   [[-1, 14], 1, Concat, [1]], #23 80*80
   [-1, 3, C3, [512, False]], #24 80*80

   [-1, 1, Conv, [512, 3, 2]],
   [[-1, 10], 1, Concat, [1]],  # cat head P5
   [-1, 3, C3, [1024, False]],  # 23 (P5/32-large)

   [[21, 24, 27,30], 1, Detect, [nc, anchors]],  # Detect(P3, P4, P5)
  ]
```

run
```
python train.py --data data.yaml --epochs 10 --batch-size -1 --cfg small.yaml
```
and
```
python train.py --data data.yaml --epochs 10 --batch-size -1
```

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by StudyingLover** [comment]
To be more precise, the loss on the validation set is empty.

**Comment by StudyingLover** [comment]
After checking, we found that there was a problem with the autodl.com mirror. Delete the code and re-git clone to solve the problem.

**Comment by glenn-jocher** [comment]
@StudyingLover thank you for bringing this issue to our attention. We apologize for any inconvenience you may have experienced. 

Based on the information you provided, it seems that there was an issue with the `autodl.com` mirror. To address this problem, we recommend deleting the existing code and re-cloning the repository to ensure a fresh copy. 

Please follow these steps to resolve the problem:
1. Delete the current YOLOv5 code from your system.
2. Clone the YOLOv5 repository again using the following command:
   ```
   git clone https://github.com/ultralytics/yolov5
   ```

This should provide you with a clean copy of the YOLOv5 codebase and resolve the issue you encountered with the loss on the validation set being empty.

If you have any further questions or need additional assistance, please feel free to ask.
